code/mk_figuresnstats.py

```diff
@@ -862,13 +866,13 @@ def savegaze():
             show_vels=True,
             coord_lim=(0, 1280),
             vel_lim=(0, 1000))
```

Comment:
```suggestion
            vel_lim=(0.001, 1000))
```

While we are at it, we could set the lower bounds of the y-axis to not exactly zero. The data is logscaled, and this results in 

```
/home/adina/repos/papers/paper-remodnav/remodnav/remodnav/tests/utils.py:124: UserWarning: Attempted to set non-positive bottom ylim on a log-scaled axis.
Invalid limit will be ignored.
  ax2.set_ylim(vel_lim)
```